packages/waku/src/router/client.ts

```diff
@@ -31,6 +31,11 @@ import {
   LOCATION_ID,
 } from './common.js';
 import type { RouteProps, ShouldSkip } from './common.js';
+import type { RouteConfig } from './base-types.js';
+
+export type InferredPaths = RouteConfig extends { paths: infer UserPaths }
```

Comment:
No need to export?
```suggestion
type InferredPaths = RouteConfig extends { paths: infer UserPaths }
```

Reply:
Good catch! I left this lingering by mistake after moving which file it was in